apps/Learning-Buddy/screens/AnsweringScreen.jsx

```diff
@@ -1,26 +1,28 @@
-import React, { useContext, useState } from 'react';
+import React, { useContext, useState, useEffect } from 'react';
 import { StyleSheet, Text, View, SafeAreaView, ScrollView } from 'react-native';
 import { colors } from '../config/colors';
 import { StyleSheetContext } from '../providers/StyleSheetProvider';
 import BigButton from '../components/BigButton';
-import SampleTestData from '../data/SampleTestData.json';
 import QuestionRadioGroup from '../components/QuestionRadioGroup';
+import { useNavigation } from '@react-navigation/native';
 
-export const AnsweringScreen = () => {
+
+export const AnsweringScreen = ({route}) => {
   const styles = useContext(StyleSheetContext);
+  const navigation = useNavigation();
 
   // boolean indicating all questions have been answered
   const [notAllQuestionsAnswered, setNotAllQuestionsAnswered] = useState(true);
 
   // final data to send to the next page
-  const [answerData, setAnswerData] = useState(SampleTestData);
+  const [answerData, setAnswerData] = useState([]);
 
   // check if all questions have been answered in the test allowing submission
   function verifyAllAnswered() {
     let check = false;
 
-    answerData.questions.map((question) => {
-      if (question.answer === '') {
+    answerData.map((question) => {
```

Comment:
I don't have any issue running this but should consider using every() method instead of map() since it only returns a Boolean.

Reply:
Thanks I changed the implementation to fit your advice